### ISSUE TITLE
Help to merge to Swea?

### ISSUE THREAD

**Issue by tmpm697** [fluix-dev/sway-borders#38]
I'm currently want to merge your PR, but seem it gets involve a bit out of original PR.

Can you help to create PR in this repo? https://github.com/tmpm697/Swea/issues/1

**Comment by fluix-dev** [comment]
This sounds like a copy of this repo without any new features.

**Comment by fluix-dev** [comment]
Alright, that was a bit harsh (I also missed the other PR you've already added). What is it that you're actually having trouble with?

**Comment by tmpm697** [comment]
aur package just created for a day, and it will include as many patches as possible so no duplicate.

I can't keep track with changes in this repo compare to your original PR: https://github.com/swaywm/sway/pull/5639

**Comment by fluix-dev** [comment]
> aur package just created for a day, and it will include as many patches as possible so no duplicate.

Duplicates aren't the issue, it's the fact that the package must be used by at least a few others and that your maintainer line should be correct.

> I can't keep track with changes in this repo compare to your original PR: swaywm#5639

I don't understand, can you rephrase/clarify? Perhaps take a look at the git history?

**Comment by tmpm697** [comment]
> Duplicates aren't the issue, it's the fact that the package must be used by at least a few others and that your maintainer line should be correct.

It's debatable so I won't comment, feel free to submit a de-submission request or whatever.


> I don't understand, can you rephrase/clarify? Perhaps take a look at the git history?

fair enough, guess it's just a few and reading through it won't hurt :)

**Comment by fluix-dev** [comment]
> guess it's just a few and reading through it won't hurt :)

Good luck!
